src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/fork/autocomplete.ts

```diff
@@ -73,19 +72,15 @@ export async function autocomplete(
     return [];
   }
 
-  const functionsSpecificSuggestions = await getInsideFunctionsSuggestions(
```

Comment:
Fork delegates everything to commands, I don't see any point in managing expressions here